.travis-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Originally from https://gist.github.com/domenic/ec8b0fc8ab45f39403dd, with my
+# modifications for webpack.
+
+TARGET_BRANCH="pr3__bikeshed"
+
+# Save some useful information
+REPO=`git config remote.origin.url`
+SSH_REPO=${REPO/https:\/\/github.com\//git@github.com:}
+SHA=`git rev-parse --verify HEAD`
+
+SUPER_REPO="https://$GH_TOKEN@github.com/miguelao/mediacapture-fromelement.git"
+
+git config --global user.name "Travis CI"
+git config --global user.email "miguelecasassanchez@gmail.com"
+git clone $SUPER_REPO
+
+cd mediacapture-fromelement
+git checkout $TARGET_BRANCH
+
+bikeshed spec
+
+git commit -m "Auto deploy ${SHA} to GitHub ${TARGET_BRANCH} branch" index.html
+git push --force --quiet $SUPER_REPO
```

Comment:
Several notes here:
1. you shouldn't need to force push
2. redirect the output to the push or you will leak your token if there are errors, I've had this wonderful pleasure myself

See what I do [here](https://github.com/martinthomson/i-d-template/blob/master/ghpages.mk#L94-L95).  The general strategy there is - I think - worth using:

```
git fetch -q origin gh-pages:gh-pages
git clone -q -b gh-pages . $TMP
cp "${files[@]}" $TMP
git -C $TMP commit -m "... [ci skip]" "${files[@]}"
git push
```